### PR TITLE
Chore: Fix the Pricing Rule Item Code popup to be displayed properly

### PR DESCRIPTION
**Pull request by shruti2323**
The "Pricing Rule Item Code" popup is displaying properly. Below are the snaps attached.
Update was mentioned in the task : -
**Move disable checkbox above apply on section**
PricingRule03
PricingRule02
PricingRule01

### CODE REVIEW

File: erpnext/accounts/doctype/promotional_scheme/promotional_scheme.json [hunk 1]
```diff
@@ -1,1381 +1,332 @@
...
  "creation": "2019-02-08 17:10:36.077402",
- "custom": 0,
- "docstatus": 0,
  "doctype": "DocType",
- "document_type": "",
  "editable_grid": 1,
  "engine": "InnoDB",
+ "field_order": [
+  "section_break_1",
+  "disable",
```
Comment (hrwX) [inline erpnext/accounts/doctype/promotional_scheme/promotional_scheme.json:11]: - Move Disable to the above section

**Review by hrwX — CHANGES_REQUESTED** [review]

**Comment by sahil28297** [comment]
@shruti2323 can you lebel your screenshots, like before after. Also move `disable` checkbox as the first field in the doctype to make it more consistent with rest of the doctypes, refer screenshot below.
image

File: erpnext/accounts/doctype/promotional_scheme/promotional_scheme.json (hunk 1 above)
Comment (shruti2323) [reply erpnext/accounts/doctype/promotional_scheme/promotional_scheme.json:11]: Its already moved to above section .
image

**Comment by shruti2323** [comment]
> @shruti2323 can you lebel your screenshots, like before after. Also move `disable` checkbox as the first field in the doctype to make it more consistent with rest of the doctypes, refer screenshot below.
> image

**Comment by shruti2323** [comment]
Ok Sahil .

**Pull request closed without merging**